tox.ini

```diff
@@ -110,6 +110,7 @@ deps =
     pytest-asyncio
     pytest-operator
     pytest-playwright
+    websockets<14.0 # https://github.com/juju/python-libjuju/issues/1184
```

Comment:
You could also pin juju==3.5.2.1 instead, it should be fixed there

Reply:
Updated, thanks!